noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -2421,10 +2591,18 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <!--
+    ============================================================================================================
+    ENDRINGSLOGG
+    ============================================================================================================
+    -->
+    <!-- Checks -->
+
+    <check id="ELC1">
         <title>Number of changes in endringslogg</title>
+        <description>Provides information about the number of changes found in endringslogg</description>
```

Comment:
`Provides information about the number of change log entries in endringslogg.xml`